packages/frontend/src/components/sign/v2/SignTransactionSummary.js

```diff
@@ -0,0 +1,99 @@
+import BN from 'bn.js';
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import AlertBanner from '../../common/AlertBanner';
+import FormButton from '../../common/FormButton';
+import FormButtonGroup from '../../common/FormButtonGroup';
+import Container from '../../common/styled/Container.css';
+import ConnectWithApplication from '../../login/v2/ConnectWithApplication';
+import SignTransaction from './SignTransaction';
+
+const StyledContainer = styled(Container)`
+    &&& {
+        h3 {
+            color: #272729;
+            text-align: center;
+        }
+    
+        > button {
+            &.link {
+                font-weight: normal;
+                margin: 25px auto 0 auto;
+                display: block;
+                :before {
+                    content: '+';
+                    letter-spacing: 5px;
+                }
+            }
+        }
+
+        .button-group {
+            margin-top: 25px;
+        }
+
+        .connect-with-application {
+            margin: 20px auto 30px auto;
+        }
+
+        .alert-banner {
+            margin: 20px 0;
+            border-radius: 4px;
+        }
+    }
+`;
+
+export default ({
+    transferAmount,
+    accountLocalStorageAccountId,
+    availableBalance,
+    estimatedFees,
+    onClickCancel,
+    onClickApprove,
+    onClickMoreInformation,
+    accountUrlReferrer,
+    submittingTransaction
+}) => {
+    const InsufficientBalance = availableBalance && transferAmount && new BN(availableBalance).lt(new BN(transferAmount));
```

Comment:
:nit: Capitalized var is not typical for a boolean value; suggestion maybe `isBalanceInsufficient`?

Reply:
Thanks for catching - fixing:)